src/components/input/input.ts

```diff
@@ -108,6 +107,93 @@ export class TextInput extends InputBase {
   /**
    * @private
    */
+  @Input() autocomplete: string;
+
+  /**
+   * @private
+   */
+  @Input() autocorrect: string;
+
+  /**
+   * @private
+   */
+  @Input() autocapitalize: string;
+
+  /**
+   * @private
+   */
+  get autofocus(): boolean { return this.autofocus; }
```

Comment:
Seems [they](https://github.com/angular/material2/blob/master/src/lib/input/input.ts#L180-L181) put the decorator on `set` rather than `get`, but yeah got it.